pdns/lua-recursor4.cc

```diff
@@ -54,6 +54,8 @@ int RecursorLua4::gettag(const ComboAddress& remote, const ComboAddress& local,
 #undef L
 #include "ext/luawrapper/include/LuaContext.hpp"
 
+extern __thread struct timeval g_now; // timestamp, updated (too) frequently
```

Comment:
this is asking for problems since it might get out of sync, perhaps put this in syncres.hh?